src/shared/GlobalTopBanners/RequestInstallBanner/RequestInstallBanner.tsx

```diff
@@ -14,6 +15,7 @@ import TopBanner, { saveToLocalStorage } from 'ui/TopBanner'
 const APP_INSTALL_BANNER_KEY = 'request-install-banner'
 const COPY_APP_INSTALL_STRING =
   "Hello, could you help approve the installation of the Codecov app on GitHub for our organization? Here's the link: https://github.com/apps/codecov/installations/select_target"
+const SHARE_REQUEST_METRICS_KEY = 'user_shared_request'
```

Comment:
if we have this more widespread might make sense to colocate all metrics keys in a shared file somewhere so no risk of duplicates at all.

Also in Sentry we typically use `.` for the delimiter, and `_` to represent compound words, and use plural for metrics, so `user.shared.requests` in this case.

![image](https://github.com/codecov/gazebo/assets/18689448/c2d3ed81-8de4-439e-94bc-f3d143bc00c7)

Do we also want to add `request.install` prefix or something similar?

`request_install.user.shared.requests`

Reply:
actually we don't use plural I think I just hallucinated that - sorry for the fake news lol

`request_install.user.shared.request` makes the most sense to me

Reply:
@AbhiPrasad, thoughts on moving this to our `sentry.ts` file and creating an `as const` object that contain our keys, we could also use functions to create a base key and extend the remainder of the key, something like:

```ts
 // sentry.ts
const METRIC_KEYS = {
  REQUEST_INSTALL: (key: string) => `request_install.${key}`
} as const

// random file
const handleClick = () => {
  metrics.increment(METRIC_KEYS.REQUEST_INSTALL('user.shared.request'))
}
```

Reply:
I think a function abstraction introduces a lot of complexity that we don't need right now, let's start simple with just plain strings? We can move to introduce abstraction when there are more metrics and we feel like we need to enforce naming.

```ts
// sentry/metricKey.ts
export const REQUEST_INSTALL_USER_SHARED_REQUEST = 'request_install.user.shared.request';
```

```
// random file
import { REQUEST_INSTALL_USER_SHARED_REQUEST } from 'blah/blah/sentry/metricKey.ts';

metrics.increment(REQUEST_INSTALL_USER_SHARED_REQUEST);
```

You can also write a wrapper function around `metrics.increment` and try to enforce the key names via a string union or even TS template literal types.

Reply:
I can document the current process(it's fairly straightforward). 

It's hard to define an abstraction that makes sense without seeing more of these in the codebase so I am willing to wait till then and use this approach for now.